_studio/shared/umc/codec/h265_dec/src/umc_h265_frame_info.cpp

```diff
@@ -127,7 +127,12 @@ void H265DecoderFrameInfo::EliminateErrors()
         bool bIndepSliceMissing = false;
         for (uint32_t sliceId = 1; sliceId < count; sliceId++)
         {
-            H265SliceHeader *sliceHeader = GetSlice(sliceId)->GetSliceHeader();
+            H265Slice *slice = GetSlice(sliceId);
```

Comment:
State when we have [count] slices but have no slice for this idx looks like invariant violation. It seems the thing is 
`uint32_t count = m_SliceCount;`
- we save number of slices and then use this number. But we may remove some slices (line #115). We need either adjust [count] or use [GetSliceCount] for this loop.

Reply:
Why we can't use m_SliceCount directly in loops?